CodeGen/Generators/NanoFrameworkGen/QuantityGenerator.cs

```diff
@@ -124,6 +124,9 @@ private void GenerateConversionProperties()
 ");
             foreach (var unit in _quantity.Units)
             {
+                if(unit.SkipConversionGeneration)
```

Comment:
We have to skip auto-generated properties and From methods since PluralName is the same and causing a clash